test/test-supervisor.js

```diff
@@ -55,130 +58,154 @@ describe('supervisor', function(done) {
     }
   }
 
-  function run(dir, args, expectations) {
-    it('should run [' + args.join(',') + '] from ' + dir, function(done) {
-      runSlr(dir, args, expectations, done);
+  function run(t, dir, args, expectations) {
+    t.test('should run [' + args.join(',') + '] from ' + dir, function(t) {
+      runSlr(dir, args, expectations, function() {
+        afterEach(end);
```

Comment:
The first release with it (https://github.com/tapjs/node-tap/commit/54ea80ccba75d7ec543fb4eb3d7f8e3aef7fd0a7) was 1.3.0, just over a year ago :-P